packages/mock-block-dock/dev/DevApp.tsx

```diff
@@ -1,14 +1,26 @@
 import * as React from "react";
+import { useState } from "react";
 import * as ReactDOM from "react-dom";
+
 import { TestBlock } from "./TestBlock";
 import { MockBlockDock } from "../src/MockBlockDock";
 
 const node = document.getElementById("app");
 
-const DevApp = () => (
-  <MockBlockDock>
-    <TestBlock entityId="test-entity-1" name="World" />
-  </MockBlockDock>
-);
+const DevApp = () => {
+  const [entityId, setEntityId] = useState(1);
+
+  return (
+    <>
+      <button type="button" onClick={() => setEntityId(entityId + 1)}>
```

Comment:
This change to the dev setup allowed me to consistently test for the buggy behavior being observed while switching blocks.